### PR TITLE
Integration Test for Patch

### PR DESCRIPTION
**Pull request by rkalra247**

### CODE REVIEW

File: src/api/mapping.ts [hunk 1]
```diff
@@ -111,46 +111,33 @@ mappingRouter.delete('/delete/:id', async (req, res) => {
...
+    return res.status(400)
+  }
+
   const domainList = domains.map((element: Mapping) => {
     if (element.id === req.params.id) {
-      if (req.body.domain) element.domain = req.body.domain
-      if (req.body.subDomain) element.subDomain = req.body.subDomain
-      if (req.body.port) element.port = req.body.port
-      if (req.body.ip) element.ip = req.body.ip
+      console.log('req body', req.body)
```
Comment (SahilKalra98) [inline src/api/mapping.ts]: Remove console.log

File: src/api/mapping.ts [hunk 2]
```diff
@@ -111,46 +111,33 @@ mappingRouter.delete('/delete/:id', async (req, res) => {
...
+      }
     }
     return element
   })
   setData('mappings', domainList)
+
   const updatedDomain = domains.find(
     (element: Mapping) => element.id === req.params.id
   )
-  const prodConfigApp = [...prodConfigure.apps][0]
```
Comment (SahilKalra98) [inline src/api/mapping.ts]: why was this removed?

File: src/public/client.ts [hunk 3]
```diff
@@ -66,6 +66,9 @@ class MappingItem {
       <button class="btn btn-sm btn-outline-primary edit" type="button">
         Edit
       </button>
+      <button class="btn btn-sm btn-outline-primary edit" type="button">
```
Comment (SahilKalra98) [inline src/public/client.ts]: This also can be removed

File: src/api/mapping.ts [hunk 4]
```diff
@@ -136,6 +146,10 @@ mappingRouter.patch('/edit/:id', async (req, res) => {
     apps: prodConfigApp
   }
 
+  if (process.env.NODE_ENV !== 'production') {
```
Comment (songz) [inline src/api/mapping.ts]: change to use `isProduction` from `enviroment`
